package.json

```diff
@@ -35,18 +32,19 @@
     "url": "git+https://github.com/jdiaz5513/capnp-ts.git"
   },
   "scripts": {
-    "bootstrap": "lerna bootstrap",
+    "bootstrap": "lerna bootstrap --hoist",
     "benchmark": "gulp benchmark",
     "build": "gulp build",
-    "ci": "gulp lint && gulp coverage && codecov --disable=gcov",
+    "ci": "gulp lint && gulp coverage && codecov --disable=gcov && cd js-examples && ./test.sh",
```

Comment:
I mean it's vital for the library itself to be cross-platform (and it very much is).

For development... eh. If people want to actually do dev on Windows I'll accept a patch but this isn't really worth fussing about now.